kclvm/cmd/src/test_data/cases/import_konfig_1/stdout

```diff
@@ -0,0 +1,5 @@
+main:
```

Comment:
windows test failed because of the windows newline `\r\n`

Reply:
Not for this reason.

Windows test failed because rust crate [`gag`](https://crates.io/crates/gag) allows redirecting stderr or stdout either to a file or to nothing, but it only works on unix systems.

I replaced `println!` with `writeln!` and added a parameter `writer` to `run_command` to redirect the output.